hi_IN.lproj/Front.strings

```diff
@@ -0,0 +1,189 @@
+"Failed to load file" = "फ़ाइल लोड करने में विफल";
+"Save File Failed!" = "फ़ाइल सहेजें विफल!";
+"Cannot open folder {0}" = "फ़ोल्डर नहीं खोल सकता {0}";
+
+"File content is changed by external applications. Reload content from disk ?" = "फ़ाइल सामग्री बाहरी अनुप्रयोगों द्वारा बदल दी जाती है। डिस्क से सामग्री को पुनः लोड करें?";
+"Should continue moving and overwrite existing file(s) ?" = "जारी रखना जारी रखना चाहिए और मौजूदा फ़ाइल को ओवरराइट करना चाहिए?";
+
+"Rename to \"{0}\"नाम बदलें" = "Rename to \"{0}\" ?";
+"Failed to rename from \"{0}\"से नाम बदलने में विफल"{1}\"" = "Failed to rename from \"{0}\" to \"{1}\"";
+
+"Successfully uploaded {0} images." = "Successfully uploaded {0} images.";
+"Successfully uploaded {0} images." = "सफलतापूर्वक {0} छवियों को अपलोड किया गया।";
+"Failed to upload image. Error Message:" = "छवि अपलोड करने में विफल। त्रुटि संदेश:";
+"Failed to Copy Image to Target Folder." = "फ़ोल्डर को लक्षित करने के लिए छवि कॉपी करने में विफल।";
+"Select Container Folder" = "कंटेनर फ़ोल्डर का चयन करें";
+
+"Saved" = "बचाया";
+
+/* place holder in find & replace input */
+"search text" = "खोज पाठ";
+"replace with" = "से बदलो";
+/* search option */
+"Case Sensitive" = "अक्षर संवेदनशील";
+"Whole Word" = "पूरा शब्द";
+
+"input image URL" = "इनपुट छवि यूआरएल";
+"Select image from local file" = "स्थानीय फ़ाइल से छवि का चयन करें";
+"Cannot open location <code>$1</code>, do you mean <a href='https://$3'>https://$2</a> ?" = "स्थान <कोड> $ 1 </ code> नहीं खोल सकता, क्या आपका मतलब है <a href='https://$3'> https: // $ 2 </a>?";
+
+"select a language" = "भाषा चुनें";
+"Finish Editing" = "संपादन समाप्त करें";
+
+"Revert advanced setting file to default one ?" = "डिफ़ॉल्ट सेटिंग फ़ाइल को डिफ़ॉल्ट पर वापस लाएं?";
+
+"Select Spell Check Language" = "वर्तनी जांच भाषा का चयन करें";
+"Auto Detect Language" = "ऑटो पता लगाते हैं";
+"Disable Spell Check" = "वर्तनी जाँच अक्षम करें";
+"Global Settingâ€¦" = "वैश्विक सेटिंग € |";
+"Dictionary file is missing to enable spellcheck support for %@." = "% @ के लिए वर्तनी जांच को सक्षम करने के लिए शब्दकोश फ़ाइल गुम है।";
+"Install Language Support" = "भाषा समर्थन स्थापित करें";
+"License Information" = "लाइसेंस जानकारी";
+"Download" = "डाउनलोड";
+"Install dictionary file only for Typora" = "केवल टाइपोरा के लिए शब्दकोश फ़ाइल स्थापित करें";
+"Spell Check (%@ unavailable)" = "वर्तनी जाँच (% @ अनुपलब्ध)";
+"Spell Check (%@ downloading)" = "वर्तनी जांच (% @ डाउनलोडिंग)";
+"Download dictionary file for %@" = "% @ के लिए शब्दकोश फ़ाइल डाउनलोड करें";
+"(Default spell check option)" = "(डिफ़ॉल्ट वर्तनी जांच विकल्प)";
+"Failed to load spellcheck module" = "स्पेलचेक मॉड्यूल लोड करने में विफल";
+
+"<strong>%@</strong> has been turned on." = "<strong>% @ </ strong> चालू किया गया है।";
+"You could turn it off from `View` menu." = "आप इसे 'व्यू` मेनू से बंद कर सकते हैं।";
+"The file is too large to render in Typora." = "टाइपोरा में प्रस्तुत करने के लिए फ़ाइल बहुत बड़ी है।";
+"Open inâ€¦" = "€ € | में खुला";
```

Comment:
could you correct translation for `open in…` ?

Reply:
just remove the extra symbols, i.e. `में खुलेगा` (would be 'more correct')